docs/en/observability/profiling-advanced-configuration.asciidoc

```diff
@@ -8,3 +8,5 @@ See the following sections for more information:
 * <<profiling-add-symbols, Add symbols for native frames>>: Push symbols to your cluster so you can see function names and line numbers in traces of applications written in programming languages that compile to native code (C, C++, Rust, Go, etc.).
 * <<profiling-use-a-proxy,Use a proxy>>:  Set up an HTTP proxy if your infrastructure host-agent installation needs one to reach {ecloud}.
 * <<profiling-probabilistic-profiling, Configure probabilistic profiling>>: Configure Universal Profiling Agent to run in probabilistic profiling mode.

```

Comment:
```suggestion
* <<profiling-probabilistic-profiling, Configure probabilistic profiling>>: Configure the Universal Profiling agent to run in probabilistic profiling mode.

```
I know this is out of your PR, but adding for consistency with your new list item.